src/renderer/entities/asset/lib/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const enum AssetsPageView {
```

Comment:
It's even named `AssetsPage` it can't reside in `entities` 😉
I suggest to move into AssetsSettings (where it's actually used), rename it in order to make assets settings feature more versatile and not tied with Assets page that much (through the name)
Name options: `AssetsListView / AssetsView / AssetsGroup / AssetViewMode / AssetsDisplay`

Reply:
Also, we store const enums in `types`
If you're used to plain objects instead of const enums, please let me know
I'm eager to replace them but stumble upon appropriate typing 

(Paths object + type is actually made this way, but I don't like holding type in constant...)
`export type PathType = ObjectValues<typeof Paths>;`
And use it like this:
```javascript
const x: Record<PathType, string> = {
  [Paths.ROOT]: '1223',
};
console.log(x);
```

Reply:
I suppose I will need it later in other features, for example, to check in the modal if I should render the page based on `assetsView` param

Also, I saw in the project in some places an enum in constants files, is it a mistake then? Is it correct to place it in types if we use enum as values?

Reply:
I guess when we don't care about keys - an enum is an optimal solution, otherwise a plain object might be better but then we have to deal with types, which is not always convenient.